ext/http/http_next.rs

```diff
@@ -397,24 +400,22 @@ pub fn op_http_set_response_header(
   resp_headers.append(name, value);
 }
 
-#[op(v8)]
-fn op_http_set_response_headers(
+#[op2]
+pub fn op_http_set_response_headers(
   scope: &mut v8::HandleScope,
```

Comment:
This `scope` isn't used, which should allow this method to become fast.

I'll file a bug -- we should be getting a warning that this parameter is unused. I suspect our `#[allow(unused)]` is a bit liberal.

Reply:
I' m not sure I understand - the scope parameter is used 5 times in a loop so I believe this is okay?

Reply:
@bartlomieju I missed the collapsed section and thought this was a bug. Resolved, PR looks good to me.